float32 current_yaw
float32 current_pitch
float32 current_distance
float32 current_latitude
float32 current_longitude
float32 current_mag_yaw
